src/main/openapi/openapi.yaml

```diff
@@ -538,19 +540,19 @@ components:
           description: 'Human readable version of the edition'
           type: string
         titleUUID:
-          description: 'TODO: Explain this'
+          description: 'The ID for the title of the newspaper, which the article is from.'
           type: string
         familyId:
-          description: 'TODO: Explain this'
+          description: 'The general name of the newspaper. The name of newspapers can change over time, this familyId will always be the same eventhough the title of the newspaper changes a little'
           type: string
         newspaper_page:
           description: 'The page number of the addition that the article appears on'
```

Comment:
`edition`

Reply:
I am not quite sure where  you want edition to be mentioned in this line? Can you please elaborate?